### ISSUE TITLE
Programatically supplying superstatic configuration

### ISSUE THREAD

**Issue by rstacruz** [firebase/superstatic#179]
I used to be able to pass an absolute path to `superstatic()`'s `config.root` back in v1.2.3:

```js
app = superstatic({
  config: { root: '/path/to/foo/bar' },
  debug: false
})

app.listen(3000, function () {
  console.log('listening')
})
```

how do i do this with the newer 2.x version? it seems to expect a relative path now.

**Comment by scottcorgan** [comment]
The same way. https://github.com/divshot/superstatic#superstaticoptions.

Does that help?

**Comment by scottcorgan** [comment]
@rstacruz are you still having trouble?

**Comment by rstacruz** [comment]
Seems to work fine! Not sure what I ran into really..

**Comment by rstacruz** [comment]
ah, right. this will no longer work:

```js
superstatic({
  config: {
    root: '/path/to/foo/bar',
    clean_urls: true
  }
})
```

but this will:

```js
superstatic({
  config: JSON.stringify({
    root: '/path/to/foo/bar',
    clean_urls: true
  })
})
```

**Comment by mbleigh** [comment]
Huh, weird. That seems like a bug, will take a look.

**Comment by mbleigh** [comment]
This will be fixed in 4.0

**Comment by mbleigh** [comment]
Fixed in 4.0!
